modules/reitit-spec/src/reitit/coercion/spec.cljc

```diff
@@ -108,12 +108,14 @@
                   (when (:body parameters)
                     {:requestBody (openapi/openapi-spec
                                    {::openapi/content (zipmap content-types (repeat (:body parameters)))})})
-                  (when (:request parameters)
+                  (when request
                     {:requestBody (openapi/openapi-spec
                                    {::openapi/content (merge
-                                                       (when-let [default (get-in parameters [:request :body])]
+                                                       (when-let [default (:body request)]
```

Comment:
shouldn't this be (get-in request [:content :default])?

Reply:
should be. for symmetry, should `[:request :body]` be supported as 2nd fallback?

1. `[:request content-type :schema]`
2. `[:request :default :schema]`
3. `[:request :body]`
4. `[:parameters :body]`

we can drop 3 too.

Reply:
I'd personally drop 3, but as long as its documented & tested it's fine to keep it too.

Reply:
keeping 3